controllers/controller.js

```diff
@@ -43,8 +43,10 @@ exports.postCommentsByReviewId = (req,res,next) => {
   const username = req.body.username;
   const review_id = req.params.review_id;
   const date = Date.now()
-  console.log(date)
-  return insertIntoCommentsByReviewId(body, username, new Date(date), review_id)
+  return selectReviewsById(review_id)
```

Comment:
You shouldn't need to do this at all actually. If you try to insert a comment with a `review_id` that is not in the `reviews` table, it will violate the FOREIGN KEY constraint - the whole reason we're utilising a relation database.
This is an unnecessary call to the database and therefore wasted computational expense.